From shemminger@linux-foundation.org  Sat May 19 20:02:02 2007
Date: Tue, 8 May 2007 13:41:49 -0700
From: Stephen Hemminger <shemminger@linux-foundation.org>
To: Greg KH <gregkh@suse.de>
Subject: Driver core: kill unused code
Message-ID: <20070508134149.055cbd65@localhost>

From: Stephen Hemminger <shemminger@linux-foundation.org>


  CC      drivers/base/dd.o
drivers/base/dd.c:211: warning: =E2=80=98device_probe_drivers=E2=80=99 defi=
ned but not used

Looks like the following is dead.

Signed-off-by: Stephen Hemminger <shemminger@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/base/dd.c |   13 -------------
 1 file changed, 13 deletions(-)

--- a/drivers/base/dd.c
+++ b/drivers/base/dd.c
@@ -207,19 +207,6 @@ static int __device_attach(struct device
 	return driver_probe_device(drv, dev);
 }
 
-static int device_probe_drivers(void *data)
-{
-	struct device *dev = data;
-	int ret = 0;
-
-	if (dev->bus) {
-		down(&dev->sem);
-		ret = bus_for_each_drv(dev->bus, NULL, dev, __device_attach);
-		up(&dev->sem);
-	}
-	return ret;
-}
-
 /**
  *	device_attach - try to attach device to a driver.
  *	@dev:	device.
